src/mtl.rs

```diff
@@ -69,6 +70,37 @@ impl Material {
     }
 }
 
+/// Indicates type of a missing value
+#[derive(Debug)]
+pub enum MtlMissingType {
+    /// i32
+    Tyi32,
```

Comment:
Why Do we need `Ty` prefix?

Reply:
We don't. This was caused by two issues on my end.

 - I forgot that enums generally have their variants PascalCase.
 - I thought that `i32` would be an invalid variant. It isn't, but it doesn't matter because point 1 :)